src/core/model.js

```diff
@@ -455,4 +458,17 @@ export default class Model extends Object3D {
   setTextures(textures = []) {
     throw new Error('model.setTextures replaced: setUniforms({sampler2D: new Texture2D})');
   }
+
+  isPickable() {
```

Comment:
It is used but in deprecated code.
https://github.com/uber/luma.gl/search?utf8=%E2%9C%93&q=isPickable&type=

Also there is a picking example that uses this (the one that shows the different planets)

Agree we should review this, but I think it would add too much to this already big PR.